src/main/java/org/jenkinsci/plugins/workflow/multibranch/WorkflowBranchProjectFactory.java

```diff
@@ -82,6 +98,17 @@ public boolean equals(Object obj) {
 
     @Extension public static class DescriptorImpl extends AbstractWorkflowBranchProjectFactoryDescriptor {
 
+        private String scriptPath = SCRIPT;
+
+        public DescriptorImpl(){ this.scriptPath = SCRIPT; }
+
+        @DataBoundConstructor
```

Comment:
No, a `Descriptor` is not a `Describable` and so it must not have a `@DataBoundConstructor`.